### ISSUE TITLE
Mellomhjernen

### ISSUE THREAD

**Issue by anjamaa** [realfagstermer/realfagstermer#1237]
Skal vi legge til "Diencephalon" på norsk også?

---
*REAL007622 (Mellomhjernen) i [Emnesøk](https://app.uio.no/ub/emnesok/realfagstermer/search?id=c007622), [Skosmos](http://data.ub.uio.no/realfagstermer/c007622), [Okapi](http://ub-viz01.uio.no/okapi2/#/search?q=realfagstermer%3A%22Mellomhjernen%22), [Soksed](http://ub-soksed.uio.no/concepts/REAL007622)*

**Comment by kyrretl** [comment]
Etter godkjenning ligger nå Diencephalon som EN og er dermed søkbar, men vi kan legge inn BF "Diencefalon" på norsk (med f istedet for ph) ref. Store norske: https://sml.snl.no/mellomhjernen

**Comment by anjamaa** [comment]
Ja! Glemte helt at vi bruker f på norsk siden slidesene i undervisningen vår som regel har termene på latin, hehe.
